.coafile

```diff
@@ -0,0 +1,14 @@
+[all]  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/.coafile
+++ b/.coafile
@@ -1,4 +1,4 @@
-[all]  
+[all]
 bears = JSONFormatBear
 indent_size = 2
 files = **.json 
```